riscv/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+- Improved assembly macro handling in asm.rs
+- 
```

Comment:
Sorry! just noticed that this is not within the `Added` subsection. Please, move it just below the "Add `miselect` CSR" line.